jetty-xml/src/main/java/org/eclipse/jetty/xml/XmlConfiguration.java

```diff
@@ -331,6 +332,29 @@ public void initializeDefaults(Object object)
     {
     }
 
+    /**
+     * Utility method to resolve a provided path against a base directory.
+     *
+     * @param baseDir the base directory
+     * @param destPath the destination path (can be relative or absolute, syntax depends on OS + FileSystem in use)
+     * @return String to resolved and normalized path, or null if baseDir or destPath is empty.
+     */
+    public static String resolvePath(String baseDir, String destPath)
```

Comment:
problem with calling it `baseDir` is that it might get confused with `jetty-base`.  But I can't think of a better name other than perhaps just `directory`

Reply:
Changed name (and javadoc) to eliminate "base"